docs/samples/advanced/derived-axis-type.md

```diff
@@ -48,3 +48,8 @@ module.exports = {
 ## Log2 axis implementation
 
 <<< @/docs/scripts/log2.js
+
+## Docs
+* [Data structures (`labels`)](../../general/data-structures.html)
```

Comment:
Particularly I pointed out `labels` every time because personally I find using month names as labels misleading for me as a user looking for ideas working with Date data.